src/rhsmlib/services/register.py

```diff
@@ -44,6 +45,16 @@ def register(self, org, activation_keys=None, environment=None, force=None, name
         if kwargs:
             raise exceptions.ValidationError(_("Unknown arguments: %s") % kwargs.keys())
 
+        if no_insights is True:
+            try:
```

Comment:
I would like to see here at least some debug print, that we try to mask `insights-register.path`.

It would be also good to add here some comment with explanation, because it's clear now, what are we trying achieve here, but it will be less clear after few years. ;-)

Reply:
Done